lib/api/helpers.rb

```diff
@@ -55,6 +55,36 @@ def find_project(id)
       end
     end
 
+    def project_issue
+      begin
+        @issue ||= user_project.issues.find(params[:issue_id])
+      rescue
+        @issue ||= user_project.issues.find_by(iid: params[:issue_id])
+      end
+
+      @issue || not_found!("Issue")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.